src/database.js

```diff
@@ -250,16 +256,14 @@ Database.prototype.batchTransaction = function(identifier) {
  */
 Database.prototype.close = function(callback) {
   var key = this.id.split('/').pop();
-  var leakError = null;
-  var leaks = this.pool_.getLeaks();
-
-  if (leaks.length) {
-    leakError = new Error(`${leaks.length} session leak(s) found.`);
-    leakError.messages = leaks;
-  }
-
   this.parent.databases_.delete(key);
-  this.pool_.close().then(() => callback(leakError), callback);
+  this.pool_.close().then(count => {
+    let error = null;
+    if (count > 0) {
+      error = new Error(`${count} session leak(s) found.`);
```

Comment:
Is it possible to track the call stacks for the leaks in question?

Reply:
👍  Should be simple to re-add this trace map setup for use here.

Reply:
I have added this functionality.